stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -47,6 +56,11 @@ public type Client client object {
     # + recordType - Type of the returned table
     # + parameters - The parameters to be passed to the select query
     # + return - A `table` returned by the SQL query statement else `Error` will be returned if there is an error
+    #
+    # # Deprecated
+    # New experimental jdbc2 module is introduced in 1.2 release and it will be fully supported with in 1.3 release
```

Comment:
```suggestion
    # A new experimental JDBC2 module is introduced in the Ballerina 1.2 release and it will be fully supported by the 1.3 release.
```